apps/engmanagement/package.json

```diff
@@ -128,7 +127,6 @@
     "next-images": "^1.8.4",
     "postcss-preset-env": "^7.4.2",
     "prettier": "^2.6.2",
-    "prisma": "^3.15.2",
```

Comment:
no need for any prisma stuff in individual packages/apps, just import `@skillrecordings/database`